### PR TITLE
[PBW-3474] Fixing the Live indicator

### PR DESCRIPTION
**Pull request by chibzu**

### CODE REVIEW

**Comment by chibzu** [comment]
Goes along with
https://github.com/ooyala/skin-config/pull/81

File: js/components/controlBar.js [hunk 1]
```diff
@@ -245,9 +245,11 @@ var ControlBar = React.createClass({
         </span>
       </div>,
 
-      "live": <div className="live" style={InlineStyle.controlBarStyle.liveItemStyle}>
-        <div style={InlineStyle.controlBarStyle.liveCircleStyle}></div>
-        <div style={InlineStyle.controlBarStyle.liveTextStyle}>{liveText}</div>
+      "live": <div className="live" style={InlineStyle.controlBarStyle.controlBarItemSetting}>
+        <div style={InlineStyle.controlBarStyle.liveItemStyle}>
+          <div style={InlineStyle.controlBarStyle.liveCircleStyle}></div>
+          <span style={InlineStyle.controlBarStyle.liveTextStyle}> {liveText}</span>
```
Comment (aldigun) [inline js/components/controlBar.js]: liveText should be using localization text file instead of hard-coded in line 239

Comment (chibzu) [inline js/components/controlBar.js]: Good catch.

**Comment by aldigun** [comment]
LGTM

**Pull request merged**